deploy/frontend.setup

```diff
@@ -0,0 +1,50 @@
+from setuptools import setup, find_packages
+
+install_requires = [
+    "pyzmq>=15.3.0",
+    "msgpack >= 1.0.0",
+    "Werkzeug >= 1.0.0",
+    "PyYAML>=3.11",
+    "Flask >= 1.1.0",
+    "Flask-Mail >= 0.9.1",
+    "pymongo>=3.2.2",
+    "oauth2>=1.9.0.post1",
+
+    #"docker>=2.5.0",
+    #"docutils>=0.14",
```

Comment:
Except if it is a requirement of another lib, docutils is required.

Reply:
Removed in https://github.com/UCL-INGI/INGInious/pull/882/commits/13e381ef3da8a4cc068bcf455fc441f3f4a671da.